src/rgw/rgw_aio.h

```diff
@@ -75,6 +83,13 @@ class Aio {
 
   // wait for all outstanding completions and return their results
   virtual AioResultList drain() = 0;
+
+  // Use unique pointer here because while the current interface may
+  // wait on a condition variable, future non-blocking versions may
+  // well *not* call `aio_operate` before returning, so
+  // ObjectOperations on the stack are unsafe.
+  static OpFunc librados_op(librados::ObjectReadOperation&& op);
```

Comment:
`Use unique pointer here` is this documentation still relevant, or did it predate the move semantics of librados::Object*Operation?

Reply:
It predated the move. Fixed, now.